src/jobBundle/Service/JobService.php

```diff
@@ -48,5 +48,80 @@ public function addJob(Form $form)
         }
         return $success;
     }
+    
+     /**
+     * @param Form $form
+     * @return bool
+     */
+    public function getJob(Form $form)
+    {
+        $success = false;
+        if($form->isSubmitted() && $form->isValid()){
+            $this->em->getRepository(jobEntity::class)->findBy(['name' => $form['name']->getData()]);
+            $success = true;
+        }
+        return $success;
+    }
+    
+     public function getJobId(Form $form)
```

Comment:
This method is not needed at all.
Intead use getJobByName method and return the job itself and extract the id or whatever is needed.